### PR TITLE
Add installation note for Alpine

### PR DESCRIPTION
**Pull request by brettlangdon**
## Description
Add a note about necessary build dependencies for installing on Alpine.

Helps (but does not resolve) #2247 and #2262

### CODE REVIEW

**Review by Yun-Kim — APPROVED** [review]

File: docs/installation_quickstart.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ your environment is ready to receive traces.
 Installation
 ------------
 
-Install with :code:`pip`::
+Install with :code:`bash`::
```
Comment (Kyle-Verhoog) [inline docs/installation_quickstart.rst:16]: pip and bash? lol the snippet isn't bash specific - will also work with any unix-y shell

Comment (brettlangdon) [reply docs/installation_quickstart.rst:16]: not sure why it was "pip" before, if this is for code highlighting "pip" isn't a language

Comment (Kyle-Verhoog) [reply docs/installation_quickstart.rst:16]: oh wait it's the language directive lol...

Comment (Kyle-Verhoog) [reply docs/installation_quickstart.rst:16]: didn't even notice

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**